src/rate/filters.py

```diff
@@ -0,0 +1,11 @@
+import django_filters
+
+from rate.models import Rate
+
+
+class RateFilter(django_filters.FilterSet):
+    date = django_filters.NumberFilter(field_name='created', lookup_expr='month')
```

Comment:
я думаю здесь `lookup_expr` должно быть `lookup_expr='date'`

Reply:
Я поставил просто для пробы по месяца, а если фильтровать по числам, то надо указывать
```suggestion
    lookup_expr='day')
```
иначе если написать 'date' выдает ошибку

Reply:
Я поставил просто для пробы по месяца, а если фильтровать по числам, то надо указывать lookup_expr='day', при написании 'date' выдает ошибку

Reply:
> я думаю здесь `lookup_expr` должно быть `lookup_expr='date'`
Я поставил просто для пробы по месяца, а если фильтровать по числам, то надо указывать lookup_expr='day', при написании 'date' выдает ошибку

Reply:
> я думаю здесь `lookup_expr` должно быть `lookup_expr='date'`

Я поставил просто для пробы по месяца, а если фильтровать по числам, то надо указывать lookup_expr='day', при написании 'date' выдает ошибку